api/src/auth/api_key_auth.py

```diff
@@ -18,17 +18,17 @@
 
 
 @dataclass
-class User:
+class ApiKeyUser:
```

Comment:
I renamed this for now to avoid it having the same name as our User table objects which I found confusing. We'll likely redo a lot of this whenever we re-implement the API key auth - this'll probably just get replaced with something like an organization.